site/ipam.md

```diff
@@ -89,21 +88,23 @@ through three states: 'deferred', 'waiting' and 'achieved':
   themselves successfully
 * 'achieved' - consensus achieved; allocations proceed normally
 
-Finally, some (but never all) peers can be launched as consensus
-observers rather than participants by specifying the `--observer`
-option:
+#### <a name="observer"></a>Via Observation
 
-    host4$ weave launch --observer $HOST3
+Finally, some (but never all) peers can be launched as observers
+by specifying the `--ipalloc-init observer` option:
 
-You do not need to specify an initial peer count to such peers. This
-can be useful to add ephemeral peers to an existing fixed cluster (for
-example in response to a scale-out event) without worrying about
-adjusting initial peer counts accordingly.
+    host4$ weave launch --ipalloc-init observer $HOST3
 
-####<a name="quorum"></a> `--init-peer-count` and How Quorum is Achieved
+You do not need to specify an initial peer count or seed to such
+peers. This can be useful to add ephemeral peers to an existing fixed
```

Comment:
Can we lose the word "ephemeral"?  It's just as useful when the peers last a long time.